woocommerce/woocommerce
# Add new `@woocommerce/product-editor` JS package

### All Submissions:






### Changes proposed in this Pull Request:

This adds the boilerplate code for a new package: `@woocommerce/product-editor` with two initial components moved over from `@woocommerce/components`: `ProductSectionLayout`, `ProductFieldSection`. 
Some more product specific components will be moved over.

Partially addresses #36355  .






### How to test the changes in this Pull Request:



1. Load this branch and run `pnpm i && pnpm run build` and make sure everything passes correctly.
2. Enable product editor at WooCommerce -> Settings -> Advanced -> Features -> Try the new product editor
3. Go to product editor at Products -> Add new (MVP)
4. Make sure all the sections still load correctly.
5. Open your console and check if `window.wc.productEditor` exists with two __experimental components.



### Other information:




### FOR PR REVIEWER ONLY:
